bftengine/include/bftengine/IRequestHandler.hpp

```diff
@@ -48,6 +48,7 @@ class IRequestsHandler {
     uint32_t maxReplySize = 0;
     char *outReply = nullptr;
     uint64_t requestSequenceNum = executionSequenceNum;
+    uint16_t reqIndexInBatch = 0;
```

Comment:
if this field is used by the pre-process, please rename it to reflect it.

Reply:
It's passed through the consensus and finally used in ReplicaImp as an input to ClientsManager.
Maybe would be clearer if I rename it to reqIndexIn**Client**Batch